### ISSUE TITLE
Nonexistent dependency in scrooge-maven-plugin

### ISSUE THREAD

**Issue by richid** [twitter/scrooge#184]
The `scrooge-generator` version was bumped to `20150406130043-4945435` in https://github.com/twitter/scrooge/commit/08929dd5dc5549f816f552360da41a3d7b8f53d0 but this artifact does not exist in Maven central or anywhere else I checked. Further, the plugin version was not revved so new installs of version `3.17.0` of this plugin will fail due to a missing dependency.

**Comment by adriancole** [comment]
got this, too, even setting twitter's repo

```
[ERROR] Failed to execute goal com.twitter:scrooge-maven-plugin:3.17.0:compile (thrift-sources) on project alerting: Execution thrift-sources of goal com.twitter:scrooge-maven-plugin:3.17.0:compile failed: Plugin com.twitter:scrooge-maven-plugin:3.17.0 or one of its dependencies could not be resolved: Could not find artifact com.twitter:scrooge-generator_2.10:jar:20150406130043-4945435 in twitter (http://maven.twttr.com) -> [Help 1]
```

**Comment by nshkrob** [comment]
Yep, this is a problem, thanks for reporting. We should release a fixed version soon.

**Comment by isnotinvain** [comment]
Related parquet jira: https://issues.apache.org/jira/browse/PARQUET-268

**Comment by maasg** [comment]
The issue is here: https://github.com/twitter/scrooge/commit/08929dd5dc5549f816f552360da41a3d7b8f53d0#diff-83adb9f8b7e12e795e5c6fab201a92bcR245

**Comment by nshkrob** [comment]
We've fixed scrooge-maven-plugin 3.17.0 on maven.org. Sorry about the breakage. Please let us know if this is still a problem.
